libs/hash/hashLib.c

```diff
@@ -0,0 +1,17 @@
+#include "openssl/crypto.h"
+#include "openssl/sha.h"
+#include "definitions/hashLib.h"
+
+void calcHash(int val, unsigned char hash[HASH_SIZE])
+{
+  SHA256((unsigned char *)&val, sizeof(val), hash);
+}
+
+void printHash(unsigned char hash[])
```

Comment:
suggestion: You created a type hash, maybe you should try using it

Reply:
fixed at [db63c5d](https://github.com/jowjow22/simplified-c-blockchain/pull/5/commits/db63c5d99982e1a4d2423f845fa9488f2f06a8ac)